packages/core/package.json

```diff
@@ -8,6 +8,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "find-up": "5.0.0"
+    "find-up": "5.0.0",
```

Comment:
Is the dependency added manually? I ask this because dependencies starting with `@` should precede others